userconfig.sh

```diff
@@ -141,6 +148,15 @@ with open("$SWAN_ENV_FILE", "w") as termEnvFile:
         if key == "PYTHONPATH":
             val = re.sub('/usr/local/lib/swan/(extensions/)?:', '', val)
         termEnvFile.write("export %s=\"%s\"\n" % (key, val))
+# Write an enviornment file to be read by the provisioner
+# for folders without a defined environment
+with open("$SWAN_DEFAULT_ENV_FILE", "w") as defaultEnvFile:
+    for key, val in dict(os.environ).items():
+        if key == "SUDO_COMMAND":
+            continue
+        if key == "PYTHONPATH":
```

Comment:
This env file is going to be used by the kernels as well, right? Please note that you're not setting the same environment as before. We removed this extra path from the file that is loaded from the terminal, but it is supposed to exist in the kernel (otherwise, the spark stuff will break).